scene/3d/camera_3d.cpp

```diff
@@ -591,7 +591,7 @@ void Camera3D::set_fov(real_t p_fov) {
 }
 
 void Camera3D::set_size(real_t p_size) {
-	ERR_FAIL_COND(p_size < 0.001 || p_size > 16384);
+	ERR_FAIL_COND(p_size <= 0.0);
```

Comment:
As a sanity check, we may want to make the lower limit be `CMP_EPSILON` instead of `0.0`.